### DISCUSSION TITLE
Setting Yazi to be my default file manager?

### DISCUSSION THREAD

**Discussion by poperigby** [sxyazi/yazi#1119] [1. Q&A] [answered]
I'm trying to setup Yazi to be my default file manager, but I'm running into some trouble.

When I press the `Show in folder` button on downloads in Firefox, I get this error in the log:

```
Failed to query file manager via ShowItems: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.FileManager1 was not provided by any .service files
```

I've set Yazi to be the default XDG MimeType handler for `inode/directory`.

**Comment by sxyazi** [comment, marked as answer]
https://github.com/GermainZ/xdg-desktop-portal-termfilechooser should be what you are looking for

**Reply by poperigby** [reply]
Thanks!
